### DISCUSSION TITLE
https://xaviw.github.io/XaviDocs/%E5%B7%A5%E5%85%B7%E7%B3%BB%E5%88%97/VitePress%E6%90%AD%E5%BB%BA/%E8%87%AA%E5%8A%A8%E7%94%9F%E6%88%90%E7%9B%AE%E5%BD%95.html

### DISCUSSION THREAD

**Discussion by giscus** [Xaviw/XaviDocs#49] [Announcements]
# https://xaviw.github.io/XaviDocs/%E5%B7%A5%E5%85%B7%E7%B3%BB%E5%88%97/VitePress%E6%90%AD%E5%BB%BA/%E8%87%AA%E5%8A%A8%E7%94%9F%E6%88%90%E7%9B%AE%E5%BD%95.html

个人前端技术文档

https://xaviw.github.io/XaviDocs/%E5%B7%A5%E5%85%B7%E7%B3%BB%E5%88%97/VitePress%E6%90%AD%E5%BB%BA/%E8%87%AA%E5%8A%A8%E7%94%9F%E6%88%90%E7%9B%AE%E5%BD%95.html

**Comment by 828767** [comment]
不知道哪里操作不对，插件启用没见生成sidebar，目录结构必须是2级吗？不想要那么多层目录的话一级目录下的md不会生成侧边？
```
- A
  - a1.md
  - a2.md
```

我测试的按默认的vitepress结构在docs目录下，设置 `srcDir: /docs` 后导航也没有了，以下是我测试的目录结构，请大佬指正：
```
├─.github
│  └─workflows
├─docs
│  ├─.vitepress
│  │  ├─cache
│  │  │  ├─deps
│  │  │  ├─deps_temp_58bfe448
│  │  │  └─deps_temp_936f0994
│  │  └─theme
│  ├─Examples
│  ├─test
│  └─public
└─node_modules
```
